packages/at_secondary_server/lib/src/verb/handler/otp_verb_handler.dart

```diff
@@ -31,10 +31,11 @@ class OtpVerbHandler extends AbstractVerbHandler {
       HashMap<String, String?> verbParams,
       InboundConnection atConnection) async {
     final operation = verbParams['operation'];
-    if (verbParams[AtConstants.ttl] != null &&
-        verbParams[AtConstants.ttl]!.isNotEmpty) {
-      otpExpiryInMills = int.parse(verbParams[AtConstants.ttl]!);
-    }
+    // int.tryParse() converts the string to an integer and sets to otpExpiryInMills.
+    // If the conversion fails, "tryParse()" returns null in which case defaultOtpExpiryInMills is set to otpExpiryInMills;
```

Comment:
Simplify the code comment here e.g. "Extract the ttl from the verb parameters if supplied, or use the default value if not"